src/components/task-step/ends.cjsx

```diff
@@ -86,6 +96,7 @@ HomeworkEnd = React.createClass
           goToStep={@goToStep}
           onNextStep={@onNextStep}
           review={type}
+          ref={"task-review-#{type}"}
```

Comment:
oh...hmm...i may have been trying to make that a key actually because of warnings about keys...